tests/nightly/estimator/test_sentiment_rnn.py

```diff
@@ -101,7 +101,19 @@ def download_imdb(data_dir='/tmp/data'):
     '''
     Download and extract the IMDB dataset
     '''
-    url = ('http://ai.stanford.edu/~amaas/data/sentiment/aclImdb_v1.tar.gz')
+    # dataset from http://ai.stanford.edu/~amaas/data/sentiment/
```

Comment:
While it is good to include the full citation here, also add the information on licensing and copyrights to the README or whatever docs people read which tells them to download the data. The idea is that some of these licenses actually have consequences. For example, we don't want to let commercial users accidentally work with a non-commercial dataset. So, our goal is to make sure that any time we inform users about a dataset, we also explain what legal requirements come with that dataset as well.

Reply:
There is not license or copy right for this dataset, only citation requriement.

Reply:
Just say the copyright is whatever person or group seems to have created it. And, say that they require attribution in place of the usual license name (but link to the request)